packages/suite/src/support/messages.ts

```diff
@@ -8748,6 +8748,10 @@ export default defineMessages({
         id: 'TR_STAKE_DAYS',
         defaultMessage: '{days} days',
```

Comment:
```suggestion
        defaultMessage: '{count, plural, one {# day} few {# days} other {# days}}'
```

We should use something like

```{count, plural, one {# day} few {# days} other {# days}}```

There is `few` to support also Czech 
```{count, plural, one {# den} few {# dny} other {# dnů}}```

Reply:
Thanks! Didn't know about this feature https://github.com/trezor/trezor-suite/pull/12553/commits/a8302aae456b37cedd07ecff8b47a27a20c49e9f